rs-lib/tests/helpers.rs

```diff
@@ -22,10 +22,10 @@ pub fn run_test(file_text: &str, run_test: impl Fn(dprint_swc_ecma_ast_view::Pro
   });
 }
 
-pub fn run_test_with_module(
+pub fn run_test_with_module<'a>(
   file_path: &Path,
   file_text: &str,
-  run_test: impl Fn(&dprint_swc_ecma_ast_view::Module),
+  run_test: impl Fn(&'a dprint_swc_ecma_ast_view::Module<'a>),
```

Comment:
I have no idea why I had to add this.

Reply:
Ended up doing a workaround that makes this no longer necessary... weird though...